content/en/blog/SSI-identity-provider/index.md

```diff
@@ -0,0 +1,62 @@
+
+As adapting these changes to OIDC client applications is undoubtedly slow, we think a midway solution not requiring too many changes to the OIDC clients would be ideal for speeding up the SSI adoption. The identity provider would work as an SSI proxy in this solution, utilizing SSI agent capabilities. Instead of storing the sensitive user data in its' database, the provider would verify the user's credential data and deliver it to the client applications using the same API interfaces as traditional OIDC.
+
+## Findy Agency under Test
+
+In the summer of 2020, our team did some initial proofs-of-concept around this subject. The experiments were successful, but our technology stack [has matured](https://findy-network.github.io/blog/2021/08/11/announcing-findy-agency/) since then. We decided to rewrite the experiments on top of our latest stack and take a closer look at this topic.
+
+{{< imgproc overview Fit "625x625" >}}
+<em>Overview of the midway solution process participants</em>
+{{< /imgproc >}}
+
+Other teams have created [similar demos](https://github.com/bcgov/vc-authn-oidc/blob/main/docs/README.md) in the past but using different SSI technology stacks. Our target was to test our [Findy Agency gRPC API](https://github.com/findy-network/findy-agent-api) hands-on. Also, our web wallet's user experience is somewhat different from other SSI wallets. The web wallet can be used securely with the browser without installing mobile applications. Furthermore, the core concept of our wallet app is the chat feature, which is almost missing altogether from other SSI wallet applications. We think that the chat feature has an essential role in creating an excellent user experience for SSI wallet users.
+
+## Demo
+
+The basic setup for the demo is familiar to OIDC utilizers. The end-user uses a browser and wishes to log in to a sample web service (["issuer tool"](https://github.com/findy-network/findy-issuer-tool)). The sample service has configured an SSI-enabled identity provider (["findy-oidc-provider"](https://github.com/findy-network/findy-oidc-provider)) as a login method and displays the button "Login via credential" on its' login page. The service redirects the user to the identity provider login page with a button click.
+
+VIDEO
+
+Then the flow changes from the usual OIDC routine. The user has already acquired the needed data (an FTN - Finnish Trust Network credential) in her wallet before the login. She reads the connection invitation as a QR code from the login page to begin the DIDComm communication with the identity provider. The identity provider will then verify the user's credential and acquire the data the client application needs for the login. The rest of the flow continues as with traditional OIDC, and finally, the client application redirects the user to the protected service. The full process sequence can be found [here](https://github.com/findy-network/findy-oidc-provider#oidc-login-flow-with-didcomm) in detail.
```

Comment:
I had to open the sequence diagram. I'm wondering whether it would be so bad if it were in the piece.